### ISSUE TITLE
Mount Everest elevation missing - due to Wikidata reference?

### ISSUE THREAD

**Issue by AJKellmann** [dbpedia/extraction-framework#767]
# Issue validity
The issue still persists, here is the link: 
http://dief.tools.dbpedia.org/server/extraction/en/extract?title=Mount+Everest&revid=&format=n-triples&extractors=custom

# Error Description
When I wrote a SPARQL query to find the highest mountain in the world, I realized that Mount Everest was missing from the results. The issue appears to be due to the lack of an <http://dbpedia.org/ontology/elevation> property in the extracted data for Mount Everest.

The Wikipedia page contains the correct height, but it is referenced from Wikidata rather than being directly included in the Infobox as is common for other mountains.
<img width="368" alt="Screenshot Wikipedia" src="https://github.com/user-attachments/assets/2fd5c730-7786-4f68-ae1e-79d4e60e70be">
This might also affect other entries in Wikipedia that load their values from Wikidata instead of stating it explicitly.

# Pinpointing the source of the error

The issue was discovered in the SPARQL endpoint at http://dbpedia.org/sparql.
Here is the SPARQL query that highlights the problem:

SELECT DISTINCT ?mountain ?height
WHERE {
  ?mountain <http://dbpedia.org/ontology/elevation> ?height.
  ?mountain a <http://schema.org/Mountain>.
}
ORDER BY DESC(?height)
LIMIT 10

Expected result: Mount Everest should appear with an elevation of 8848.86 meters. Actual result: Mount Everest does not appear in the list, indicating the elevation data is missing.

# Details
Wrong triples / missing data:
There is no <http://dbpedia.org/ontology/elevation> triple for the resource Mount Everest in the current DBpedia data.

Expected corrected RDF outcome:
<http://dbpedia.org/resource/Mount_Everest> <http://dbpedia.org/ontology/elevation>  8848.86 (xsd:double)

**Comment by debx4** [comment]
I would love to work on this issue . But I just can't find the file where the data is missing .
It would be me pleasure if @AJKellmann can help .

**Comment by AJKellmann** [comment]
@debx4
The triple is missing here:
https://dbpedia.org/resource/Mount_Everest

**Comment by debx4** [comment]
I try to resolve the problem in http://dbpedia.org/sparql
I hope it should have resolve the problem

**Comment by AJKellmann** [comment]
Hi @debx4,

Thanks for following up! I checked the SPARQL endpoint again, but unfortunately, there are still no changes regarding Mount Everest.

Just to clarify, direct edits to the database aren't possible for regular users of the SPARQL endpoint (http://dbpedia.org/sparql). Do you have writing permissions for the DBpedia server? (I don't)
These kinds of changes usually require access to the Virtuoso server running DBpedia, which isn’t publicly accessible.
If I had those permissions, I’d have applied a quick fix myself.
For such updates, you'd need to log in via the Virtuoso Conductor interface (http://dbpedia.org/conductor or its internal equivalent), which is not accessible publicly.

That said, even a quick fix would only address this one case. The actual issue lies in how the extraction framework processes Wikipedia pages that use **fetchwikidata** to pull data from Wikidata. Values like the elevation for Mount Everest aren’t currently included in the RDF output. Fixing this properly would require changes to the framework.
That's why I posted this issue here in this GitHub repository.
And I'm not sure whether this would be something that is easy to fix.

**Comment by debx4** [comment]
Yeah actually I have tried that issue to solve and it have accepted my request but as you have mentioned that database isn't accessible to public so I think that's why changes are not shown.

**Comment by AJKellmann** [comment]
You're right, the SPARQL query is valid, but without write access, DBpedia won't apply the change.
The deeper issue is still how fetchwikidata values are handled during extraction.
If you want to keep trying, don’t hesitate to ask me.
We’ll keep this issue open for now.

**Comment by JyotiP24** [comment]
Hi, can you assign this to me?

**Comment by RitikaSen27** [comment]
Hi! I see this issue is assigned but hasn’t had activity recently.  
I’d like to work on it and submit a PR if that’s okay.  
Please let me know if I can take it over.
